src/components/BackButton/BackButton.vue

```diff
@@ -2,18 +2,26 @@
   <a
     v-if="useHistory && history.previousPageUrl"
     :href="history.previousPageUrl"
-    class="back-button"
     @click.prevent="history.goBack()"
+    class="button button--round"
   >
-    <BackButtonContent />
+    <SvgIcon
```

Comment:
I did this because the styling gave two different focus outlines for the buttons:
<img width="141" alt="Scherm­afbeelding 2023-05-03 om 09 45 47" src="https://user-images.githubusercontent.com/3482946/235858590-221cbad3-0711-48ba-9d47-da6236967448.png">

Also, CSS positioning should not be on the button component, but on the parent component.